### PR TITLE
add gestion of value objects

### PR DESCRIPTION
**Pull request by x0fma**

### CODE REVIEW

**Comment by xhuberty** [comment]
Attention au message des commits, et limiter le nombre de commit vu le nombre de changement.

File: src/EntityManager.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function findBy$field(\$fieldValue, \$orderBy = null, \$limit = null, \$o
 	 * @return $entityName
 	 */
 	public function findOneBy$field(\$fieldValue, \$orderBy = null) {
-		return \$this->__call('findOneBy$field', array(\$fieldValue, \$orderBy));
+		return \$this->findOneBy(array('$fieldName' => \$fieldValue), \$orderBy);
```
Comment (xhuberty) [inline src/EntityManager.php]: array(".var_export($fieldName, true)." => \$fieldValue)

File: src/EntityManager.php [hunk 2]
```diff
@@ -122,7 +122,7 @@ private function generateDAO($data, $daoPath){
 	 * @return ".$entityName."[]
 	 */
 	public function findBy$field(\$fieldValue, \$orderBy = null, \$limit = null, \$offset = null) {
-		return \$this->__call('findBy$field', array(\$fieldValue, \$orderBy, \$limit, \$offset));
+		return \$this->findBy(array('$fieldName' => \$fieldValue), \$orderBy, \$limit, \$offset);
```
Comment (xhuberty) [inline src/EntityManager.php]: array(".var_export($fieldName, true)." => \$fieldValue)

**Comment by moufmouf** [comment]
Note for @x0fma : you don't need to close a pull-request and open a new one on any change. Simply push additional commits and they will be bundled in the pull-request.

**Pull request merged**